Modules/IO/JPEG/src/itkJPEGImageIO.cxx

```diff
@@ -334,14 +338,17 @@ JPEGImageIO::ReadImageInformation()
     case 1:
       m_PixelType = IOPixelEnum::SCALAR;
       break;
-    case 2:
-      m_PixelType = IOPixelEnum::VECTOR;
-      break;
     case 3:
       m_PixelType = IOPixelEnum::RGB;
       break;
     case 4:
+      // TODO
       m_PixelType = IOPixelEnum::RGBA;
+      itkWarningMacro("JPEG image may not be opened correct");
```

Comment:
4 components JPEGs are unlikely, probably never RGBA. Mostly CMYK, rare, [here](https://data.kitware.com/api/v1/item/61bc65c34acac99f42e67a5c/download?contentDisposition=inline) is one example. Opened with this IO it looks more "pale" as in other viewers.